specification/digitaltwins/data-plane/Microsoft.DigitalTwins/stable/2020-10-31/digitaltwins.json

```diff
@@ -5,7 +5,7 @@
     "title": "Azure Digital Twins API",
     "description": "A service for managing and querying digital twins and digital twin models."
   },
-  "host": "digitaltwins-name.digitaltwins.azure.net",
+  "host": "digitaltwins-hostname",
```

Comment:
@vinagesh do we need to do anything for our test recording/playback to account for this change?